src/hipscat_import/catalog/map_reduce.py

```diff
@@ -61,18 +63,28 @@ def _iterate_input_file(
     required_columns = [ra_column, dec_column]
 
     for chunk_number, data in enumerate(file_reader.read(input_file)):
-        if not all(x in data.columns for x in required_columns):
-            raise ValueError(
-                f"Invalid column names in input file: {ra_column}, {dec_column} not in {input_file}"
+        if use_hipscat_index:
+            if data.index.name == HIPSCAT_ID_COLUMN:
```

Comment:
I don't think so? There are a few places where we might want to look for the `_hipscat_index` if the user says they want to use it. It might be the table's index, in which case, we can't use it as a normal column. Or maybe there is no hipscat index field in there at all.

Reply:
Year, I understand the code, but my stack size wasn't large enough to read it from the first attempt...